kcp/controllers/watcher_controller.go

```diff
@@ -224,23 +195,24 @@ func (r *WatcherReconciler) createOrUpdateServiceMeshConfigForCR(ctx context.Con
 	return nil
 }
 
-func (r *WatcherReconciler) createOrUpdateIstioGwForCR(ctx context.Context, istioClientSet *istioclient.Clientset,
-	namespace string,
+func (r *WatcherReconciler) createOrUpdateIstioGwForCR(ctx context.Context,
+	istioClientSet *istioclient.Clientset,
 ) error {
 	listenerGateway, apiErr := istioClientSet.NetworkingV1beta1().
-		Gateways(namespace).Get(ctx, istioGatewayResourceName, metav1.GetOptions{})
-	ready, err := util.IstioResourcesErrorCheck(istioGatewayGVR, apiErr)
+		Gateways(IstioGatewayNamespace).Get(ctx, IstioGatewayResourceName, metav1.GetOptions{})
+	ready, err := util.IstioResourcesErrorCheck(util.IstioGatewayGVR, apiErr)
 	if !ready {
 		return err
 	}
 
 	if errors.IsNotFound(apiErr) {
```

Comment:
unreachable code!

Reply:
I added this because the `lint` pipeline was complaining about the length of the method.